### ISSUE TITLE
MixedRealityToolkitInspector freezes the editor

### ISSUE THREAD

**Issue by FejZa** [XRTK/XRTK-Core#69]
## Describe the bug
XRTK's `MixedRealityToolkitInspector` will freeze the editor when the GameObject with the `MixedRealityToolkit` component attached to it is selected and active in hierarchy.

## To Reproduce
1. Edit the project manifest.json and add the XRTK
2. Launch the editor and wait for it to import XRTK via package manager
3. Configure the XRTK via context menu
4. Select the MixedRealityToolkit GameObject in hierarchy

## Expected behavior
The editor does not freeze and I can inspect the `MixedRealityToolkit` component in the inspector.

## Information:
 - Unity 2018.3.11f1
- UPM v0.1.0 release of XRTK

## Additional context
```
NullReferenceException: Object reference not set to an instance of an object
XRTK.Services.MixedRealityToolkit.EnsureMixedRealityRequirements () (at Library/PackageCache/com.xrtk.core@792b32d81829ba6b3f7327366f70ea787e06b841/Services/MixedRealityToolkit.cs:643)
XRTK.Services.MixedRealityToolkit.InitializeServiceLocator () (at Library/PackageCache/com.xrtk.core@792b32d81829ba6b3f7327366f70ea787e06b841/Services/MixedRealityToolkit.cs:352)
XRTK.Services.MixedRealityToolkit.ResetConfiguration (XRTK.Definitions.MixedRealityToolkitConfigurationProfile profile) (at Library/PackageCache/com.xrtk.core@792b32d81829ba6b3f7327366f70ea787e06b841/Services/MixedRealityToolkit.cs:115)
XRTK.Inspectors.MixedRealityToolkitInspector.OnInspectorGUI () (at Library/PackageCache/com.xrtk.core@792b32d81829ba6b3f7327366f70ea787e06b841/Inspectors/MixedRealityToolkitInspector.cs:91)
UnityEditor.InspectorWindow.DoOnInspectorGUI (System.Boolean rebuildOptimizedGUIBlock, UnityEditor.Editor editor, System.Boolean wasVisible, UnityEngine.Rect& contentRect) (at C:/buildslave/unity/build/Editor/Mono/Inspector/InspectorWindow.cs:1625)
UnityEngine.GUIUtility:ProcessEvent(Int32, IntPtr)

manifest.json
{
  "dependencies": {
    "com.unity.package-manager-ui": "2.0.7",
    "com.unity.probuilder": "4.0.4",
    "com.unity.textmeshpro": "1.3.0",
    "com.unity.xr.arcore": "1.0.0-preview.24",
    "com.unity.xr.arfoundation": "1.0.0-preview.22",
    "com.unity.xr.arkit": "1.0.0-preview.27",
    "com.unity.xr.oculus.standalone": "1.29.1",
    "com.unity.xr.windowsmr.metro": "1.0.8",
    "com.xrtk.core": "https://github.com/XRTK/XRTK-Core.git#0.1.0",
    "com.xrtk.upm-git-extension": "https://github.com/XRTK/UpmGitExtension.git#0.9.1",
    "com.unity.modules.ai": "1.0.0",
    "com.unity.modules.animation": "1.0.0",
    "com.unity.modules.assetbundle": "1.0.0",
    "com.unity.modules.audio": "1.0.0",
    "com.unity.modules.cloth": "1.0.0",
    "com.unity.modules.director": "1.0.0",
    "com.unity.modules.imageconversion": "1.0.0",
    "com.unity.modules.imgui": "1.0.0",
    "com.unity.modules.jsonserialize": "1.0.0",
    "com.unity.modules.particlesystem": "1.0.0",
    "com.unity.modules.physics": "1.0.0",
    "com.unity.modules.physics2d": "1.0.0",
    "com.unity.modules.screencapture": "1.0.0",
    "com.unity.modules.terrain": "1.0.0",
    "com.unity.modules.terrainphysics": "1.0.0",
    "com.unity.modules.tilemap": "1.0.0",
    "com.unity.modules.ui": "1.0.0",
    "com.unity.modules.uielements": "1.0.0",
    "com.unity.modules.umbra": "1.0.0",
    "com.unity.modules.unityanalytics": "1.0.0",
    "com.unity.modules.unitywebrequest": "1.0.0",
    "com.unity.modules.unitywebrequestassetbundle": "1.0.0",
    "com.unity.modules.unitywebrequestaudio": "1.0.0",
    "com.unity.modules.unitywebrequesttexture": "1.0.0",
    "com.unity.modules.unitywebrequestwww": "1.0.0",
    "com.unity.modules.vehicles": "1.0.0",
    "com.unity.modules.video": "1.0.0",
    "com.unity.modules.vr": "1.0.0",
    "com.unity.modules.wind": "1.0.0",
    "com.unity.modules.xr": "1.0.0"
  },
  "lock": {
    "com.xrtk.core": {
      "hash": "792b32d81829ba6b3f7327366f70ea787e06b841",
      "revision": "0.1.0"
    },
    "com.xrtk.upm-git-extension": {
      "hash": "9cdc5a92ad0d1f3606026765a29d458555e54251",
      "revision": "0.9.1"
    }
  }
}```

**Comment by StephenHodgson** [comment]
Thanks for reporting. The 0.1.0 release will be changing soon. The upm story isn't quite ready yet.

I'll keep you posted.

**Comment by StephenHodgson** [comment]
Closed with the 0.1.2 release
